pom.xml

```diff
@@ -1,23 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
-  <groupId>nl.knaw.dans.api</groupId>
+   <parent>
+       <groupId>nl.knaw.dans.shared</groupId>
+       <artifactId>dans-prototype</artifactId>
+       <version>1.11</version>
+   </parent>
+  <groupId>nl.knaw.dans.easy</groupId>
   <artifactId>easy-deposit</artifactId>
   <packaging>war</packaging>
-  <version>1.0-SNAPSHOT</version>
-  <name>EASY Deposit - SWORD V2.0 deposit service</name>
-  <url>http://maven.apache.org</url>
+  <version>1.x-SNAPSHOT</version>
+  <name>EASY Deposit Service</name>
+  <description>SWORD v2 based dataset deposit service</description>
+  <url>https://github.com/DANS-KNAW/easy-deposit</url>
   <dependencies>
     <dependency>
-      <groupId>junit</groupId>
```

Comment:
We should add unit-tests, but not using junit, and certainly not such an old version :)